app/presenters/citizen_readiness/link_presenter.rb

```diff
@@ -0,0 +1,30 @@
+module CitizenReadiness
+  class LinkPresenter
+    include Tracking
+
+    attr_reader :base_path
+
+    def initialize(base_path, index)
+      @base_path = base_path
+      @index = index #used in link tracking
```

Comment:
Might be worth adding the same comment to `BrexitTaxonPresenter`, as it's quite useful here and would serve the same purpose there.

Reply:
👍  done